doc/Compatibility.md

```diff
@@ -34,6 +34,10 @@ that issue, please open an issue so the workaround can be included in the new ve
 | Google Pixel 6a            | ST54J             | 13              | Stock 13.0.0 (TQ3A.230605.010)             |   y   |         y         |   y   |   y    |                                                      |
 | Google Pixel 7             | ST54K             | 15              | Stock 15.0.0 (AP4A.241205.013)             |   y   |         y         |   y   |   y    |                                                      |
 |                            | ST54K             | 14              | Stock 14.0.0 (AP1A.240405.002.B1)          |   y   |         y         |   y   |   y    |                                                      |
+| Google Pixel 6             | ST54J             | 16              | Stock 16.0.0 (BP2A.250605.031.A2)          |   y   |         y         |   y   |   y    |                                                      |
+| Google Pixel 9 Pro XL      | ST54L             | 16              | Stock 16.0.0 (BP2A.250705.008)             |   y   |         y         |   y   |   y    |                                                      |
+| Samsung Galaxy S21 5G      | NXP (SN1X0)       | 15              | Stock 15.0.0 (AP3A.240905.015.A2)          |   y   |         y         |   y   |   y    |                                                      |
+| Sony Xperia Z5 Compact     | NXP (PN547C2)     | 7.1             | LineageOS 7.1.2 (Nougat)                   |   n   |         y         |  ro   |   n    | UID emission not supported (Snapdragon 810 software NFC stack). Only works as relay reader. |
```

Comment:
Please remove `Sony Xperia Z5 Compact` from the compatibility matrix and create a new issue for the device. Let's first identify the issue and try fixing it before giving up.